modules/showheroes-bsBidAdapter.js

```diff
@@ -1,200 +1,122 @@
+      currency: 'EUR',
+      mediaType: '*',
+      size: '*',
+    });
+    if (!isNaN(floor?.floor) && floor?.currency === 'EUR') {
+      imp.bidfloor = floor.floor;
+      imp.bidfloorcur = 'EUR';
+    }
+    return imp;
+  },
+  request(buildRequest, imps, bidderRequest, context) {
+    const req = buildRequest(imps, bidderRequest, context);
+    // delete user agent from oRTB, we'll get it from the header
+    (req?.device?.ua) && delete req.device['ua'];
+    // 'sua' is 2.6 standard, we operate with 2.5
+    (req?.device?.sua) && delete req.device['sua'];
+    return req;
+  },
 
-        if (isViralize) {
-          rBid.unitId = getBidIdParameter('unitId', bid.params);
-          rBid.sizes = size;
-          rBid.mediaTypes = {
-            [type]: {'context': context}
-          };
-        } else {
-          rBid.playerId = getBidIdParameter('playerId', bid.params);
-          rBid.mediaType = type;
-          rBid.size = {
-            width: size[0],
-            height: size[1]
-          };
-          rBid.gdprConsent = gdprConsent;
-          rBid.uspConsent = uspConsent;
-        }
+  bidResponse(buildBidResponse, bid, context) {
+    const bidResponse = buildBidResponse(bid, context);
 
-        return rBid;
+    if (context.imp?.video?.ext?.context === 'outstream') {
+      const renderConfig = {
+        rendererUrl: bid.ext?.rendererConfig?.rendererUrl,
+        renderFunc: bid.ext?.rendererConfig?.renderFunc,
+        renderOptions: bid.ext?.rendererConfig?.renderOptions,
       };
-
-      if (isViralize) {
-        if (videoSizes && videoSizes[0]) {
-          adUnits.push(makeBids(VIDEO, videoSizes, isViralize));
-        }
-        if (bannerSizes && bannerSizes[0]) {
-          adUnits.push(makeBids(BANNER, bannerSizes, isViralize));
-        }
-      } else {
-        videoSizes.forEach((size) => {
-          adUnits.push(makeBids(VIDEO, size));
-        });
-
-        bannerSizes.forEach((size) => {
-          adUnits.push(makeBids(BANNER, size));
-        });
+      if (renderConfig.renderFunc && renderConfig.rendererUrl) {
+        bidResponse.renderer = createRenderer(bidResponse, renderConfig);
       }
-    });
-
-    let endpointUrl;
-    let data;
+    }
+    bidResponse.callbacks = bid.ext?.callbacks;
+    bidResponse.extra = bid.ext?.extra;
+    return bidResponse;
+  },
+})
 
-    const QA = validBidRequests[0].params.qa || {};
+const GVLID = 111;
 
-    if (isViralize) {
-      endpointUrl = VIRALIZE_ENDPOINT;
-      data = {
-        'bidRequests': adUnits,
-        'context': {
-          'gdprConsent': gdprConsent,
-          'uspConsent': uspConsent,
-          'schain': defaultSchain,
-          'pageURL': QA.pageURL || encodeURIComponent(pageURL)
-        }
-      }
-    } else {
-      endpointUrl = isStage ? STAGE_ENDPOINT : PROD_ENDPOINT;
+export const spec = {
+  code: BIDDER_CODE,
+  gvlid: GVLID,
+  aliases: ['showheroesBs'],
+  supportedMediaTypes: [VIDEO],
```

Comment:
So this PR is removing `BANNER` support from this adapter?

I feel like this could be considered a "breaking change" and might have to be put into a "major" prebid version.

Lets ask @patmmccann @bretg @dgirardi @ChrisHuie

Reply:
Currently, we don't have anyone using `showheroes` prebid bidder for banner ads. We plan to do the 'full rework' of banner ads handling next year (probably).

My initial thought was because of that is to temporarily remove the support for `BANNER` and then later open a new PR where `BANNER` is re-enabled.

But if this would be a 'breaking change' requiring us to wait for next major release, I can revert my commit and 're-enable' legacy `BANNER` support.

Reply:
let us know this ASAP, @robertrmartinez . 
In the future we'll re-add banners, but for the moment they're not working and we thoght we could save some bytes by just removing code that was unnecessary. Releasing this adapter to the OS is critical for the next set of releases. So, if to speed up the process, we must reintroduce support for a media type that was not working, we will. 

And btw, hello dear PBJS friends :)